AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -258,6 +262,22 @@ - (void)signOut {
 
 #pragma mark - Private methods
 
+- (BOOL)checkURLSchemeRegistered:(NSString *)urlScheme {
+  NSArray *schemes;
+  NSString *typeRole;
+  NSArray *types = [MS_APP_MAIN_BUNDLE objectForInfoDictionaryKey:kMSCFBundleURLTypes];
```

Comment:
I guess this will crash if it is not an `NSArray`

Reply:
Also can it be `nil`?

Reply:
It is an `NSArray`. Look at below answer.
I also added a unit test for the case if it is `nil`.